rosetta-lang/src/main/java/com/regnosys/rosetta/scoping/RosettaScopeProvider.xtend

```diff
@@ -101,32 +100,32 @@ class RosettaScopeProvider extends ImportedNamespaceAwareLocalScopeProvider {
 				}
 				case ROSETTA_FEATURE_CALL__FEATURE: {
 					if (context instanceof RosettaFeatureCall) {
-						return createExtendedFeatureScope(context.receiver, typeProvider.getRType(context.receiver))
+						return createExtendedFeatureScope(context.receiver, typeProvider.getRMetaAnnotatedType(context.receiver))
 					}
 					return IScope.NULLSCOPE
 				}
 				case ROSETTA_DEEP_FEATURE_CALL__FEATURE: {
 					if (context instanceof RosettaDeepFeatureCall) {
-						return createDeepFeatureScope(typeProvider.getRType(context.receiver))
+						return createDeepFeatureScope(typeProvider.getRMetaAnnotatedType(context.receiver).RType)
 					}
 					return IScope.NULLSCOPE
 				}
 				case CHOICE_OPERATION__ATTRIBUTES: {
 					if (context instanceof ChoiceOperation) {
-						return createExtendedFeatureScope(context.argument, typeProvider.getRType(context.argument))
+						return createExtendedFeatureScope(context.argument, typeProvider.getRMetaAnnotatedType(context.argument))
```

Comment:
Given that putting metadata features in this operation is not supported in Java (see https://github.com/finos/rune-dsl/pull/845/files#r1804937370), I would remove metadata here.

Reply:
done